augur/filter/subsample.py

```diff
@@ -298,6 +299,25 @@ def get_group_sizes(groups, target_group_size, random_seed=None):
     return max_sizes_per_group
 
 
+def get_weighted_group_sizes(groups, group_by, weights_file, target_total_size):
+    weights = pd.read_csv(weights_file, delimiter='\t')
+
+    # FIXME: check that weights file covers all `groups`
```

Comment:
_(Notes while skimming the draft PR)_

I'd expect a common usage would be a weights TSV covering a subset of the groups. E.g. provide weights for country and then `--group-by country year` or somesuch. Do you think this is reasonable to support in the v1?

Reply:
Yes, what you've described is in the current implementation with a small difference in the command line configuration.

https://github.com/nextstrain/augur/blob/adc1d0bde9ab1832197f64060a924f1f43301971/tests/functional/filter/cram/subsample-weighted.t#L48-L67

I'll look into updating to require all columns used for grouping in `--group-by` rather than just the unweighted columns - that seems more clear.

Reply:
Done as 8d72be055d1edcaf2c4a411c4757f9dc9dd655a3